examples/counter-metrics/counter_metrics.description

```diff
@@ -1,5 +1,6 @@
 // Ballerina supports Observability out of the box, and Metrics is one of the three important aspect of the
-// Observability. To observe Ballerina code, the `--b7a.observability.enabled=true` property should be given when starting the service.
+// Observability. To observe Ballerina code, the `--observability-included` option and `--b7a.observability.metrics.enabled=true`
+// property should be given when starting the service.
```

Comment:
We need to specifically mention which ones is the build time flag and runtime flag

Reply:
Please update in all BBEs

Reply:
Fixed with 0e9309f1eaf084500cf4d36a75141e8b1f1a2ce9